smaily-for-wp.php

```diff
@@ -151,16 +153,26 @@ function smaily_subscribe_callback() {
 
 	$array = array_merge( $array, $form_values );
 	require_once( BP . DS . 'code' . DS . 'Request.php' );
-	$request = new Wp_Sendsmaily_Request( $server, $array );
-	$result  = $request->exec();
+	$request = new Smaily_Plugin_Request( $server, $array );
+	$result  = $request->post();
 
 	if ( empty( $result ) ) {
-		echo esc_html__( 'Something went wrong', 'wp_sendsmaily' );
+		echo __( 'Something went wrong', 'wp_smaily' );
 	} elseif ( (int) $result['code'] !== 101 ) {
-		// Possible errors, for translation.
-		// esc_html__('Posted fields do not contain a valid email address.', 'wp_sendsmaily');.
-		// esc_html__('No autoresponder data set.', 'wp_sendsmaily');.
-		echo esc_html__( $result['message'], 'wp_sendsmaily' );
+		switch ( $result['code'] ) {
+			case 201:
+				echo __( 'Form was not sent using POST method.', 'wp_smaily' );
```

Comment:
Use "submitted" instead of "sent". You are not sending the form, you are submitting its contents.